traffic_ops/testing/api/v5/cdn_federations_test.go

```diff
@@ -220,10 +219,9 @@ func validateCDNFederationCNameSort() utils.CkReqFunc {
 	return func(t *testing.T, _ toclientlib.ReqInf, resp interface{}, alerts tc.Alerts, _ error) {
 		assert.RequireNotNil(t, resp, "Expected CDN Federation response to not be nil.")
 		var federationCNames []string
-		CDNFederationResp := resp.([]tc.CDNFederation)
-		for _, CDNFederation := range CDNFederationResp {
-			assert.RequireNotNil(t, CDNFederation.CName, "Expected CDN Federation CName to not be nil.")
```

Comment:
Why are the assert of CName and ID checks removed?

Reply:
Because those cannot be nil, so the check is useless.